examples/mnist/README.md

```diff
@@ -0,0 +1,69 @@
+# MNIST
+
+
+We show an example of building a cnn network trained on MNIST dataset.
+
+This example includes the following steps:
+
+- Loading MNIST dataset and do some very basic preprocessing
+- Building a CNN network using tensorflow keras modules
+- Sanity Check: testing the (untrained) model prediction inference on local environment.
+- Setting up Leap env
+- Integrate the model and dataset into the TL.
+
+The model achieved accuracy of 0.98~. 
+
```

Comment:
Consider adding a short section about data integration. Explaining that in this example we are reading the mnist dataset from a public Google Cloud Storage bucket.

Reply:
Not sure if it will be TMI. I reckon it's better to explain the data integration in the main folder